styles/main.css

```
@@ -2,3 +2,97 @@
 :root {
   --bs-font-sans-serif: 'Roboto', system-ui, -apple-system, "Segoe UI", "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
 }
+
+
+div {
+  display: block;
+}
+
+/********************/
+/*    Customers     */
+/********************/
+
+.blue {
+  color: #00cec9;
+}
+.img-fluid {
+  max-width: 100%;
+  height: auto;
+  margin-left: auto;
+  display: block;
+}
+
+.slider-1 {
+  padding-top: 2rem;
+  padding-bottom: 3.75rem;
+  background: linear-gradient(rgb(238, 238, 238), rgb(144, 173, 198));
+  text-align: center;
+}
+
+.slider-1 h4 {
+  margin-bottom: 0.25rem;
+}
+
+.slider-1 .section-divider {
+  margin-bottom: 1rem;
+}
+
+.slider-1 .slider-container {
+  margin-bottom: 0.5rem;
+  padding: 1rem 0;
+}
+
+.swiper-container {
+  margin: 0 auto;
+  position: relative;
+  overflow: hidden;
+  list-style: none;
+  padding: 0;
+  z-index: 1;
+}
+
+.swiper-slide {
+  -webkit-flex-shrink: 0;
+  -ms-flex-negative: 0;
+  flex-shrink: 0;
+  width: 100%;
+  height: 100%;
+  position: relative;
+  -webkit-transition-property: -webkit-transform;
+  -o-transition-property: transform;
+  transition-property: transform, -webkit-transform;
+}
+.swiper-container-android .swiper-slide, .swiper-wrapper {
+  transform: translate3d(0px, 0, 0);
+}
+
+.swiper-container-android .swiper-slide, .swiper-wrapper {
+  -webkit-transform: translate3d(0px, 0, 0);
+}
+
+.row {
+  --bs-gutter-x: 1.5rem;
+  --bs-gutter-y: 0;
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: calc(var(--bs-gutter-y) * -1);
+  margin-right: calc(var(--bs-gutter-x)/ -2);
+  margin-left: calc(var(--bs-gutter-x)/ -2);
+}
```

Try not to override Bootstrap classes and variables, just apply the changes for your components. You can also change the `margin` properties to the [summarized syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/margin#syntax)

```suggestion
.some-class-name {
  display: flex;
  flex-wrap: wrap;
  margin: 0 -0.75rem;
}
```